lib/Mojo/Server/Morbo.pm

```diff
@@ -0,0 +1,155 @@
+package Mojo::Server::Morbo;
+use Mojo::Base -base;
+
+# "Linda: With Haley's Comet out of ice, Earth is experiencing the devastating
+#         effects of sudden, intense global warming.
+#  Morbo: Morbo is pleased but sticky."
+use Mojo::Loader qw(load_class);
+use Mojo::Server::Daemon;
```

Comment:
This comment should be on script/morbo, but is it possible to also load the listen value from a config file like hypnotoad does?